apps/meteor/server/models/raw/Messages.ts

```diff
@@ -424,14 +429,25 @@ export class MessagesRaw extends BaseRaw<IMessage> implements IMessagesModel {
 		return this.find(query, options);
 	}
 
-	findLivechatMessagesWithoutClosing(rid: IRoom['_id'], options?: FindOptions<IMessage>): FindCursor<IMessage> {
-		return this.find(
-			{
-				rid,
-				t: { $exists: false },
-			},
-			options,
-		);
+	findLivechatMessagesWithoutTypes(
```

Comment:
Instead of creating a new func, can we reuse the `findVisibleByRoomIdNotContainingTypesBeforeTs` one?

Reply:
Duplicate.

Reply:
sorry im dum didn't see this was dupe lol